release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3023,6 +3023,70 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Before this update, the HAProxy configuration used the `/version` endpoint for health checks causing unreliable health checks to be generated. With this release, the liveness probe is customized to use `/livez?exclude=etcd&exclude=log` on {ibm-cloud-title} for more accurate health checks avoiding disruptions due to inappropriate probe configurations on Hypershift, while retaining `/version` for other platforms. (link:https://issues.redhat.com/browse/OCPBUGS-58126[OCPBUGS-58126])
+
+* Before this update, the Machine Config Operator (MCO) updated boot images without verifying that the current boot image was from the marketplace. As a consequence, the MCO overrode a marketplace boot image with a standard {product-title} installer image. With this release, the MCO references a lookup table in {aws-first} that contains all standard installer Amazon Machine Images (AMIs) in {product-title} before it updates the boot image. In {gcp-first}, the MCO checks the URL header before updating the boot image. As a result, the MCO does not update machine sets that have a marketplace boot image. (link:https://issues.redhat.com/browse/OCPBUGS-58044[OCPBUGS-58044])
+
+* Before this update, a validation issue in the `oc-mirror` plugin caused the command to reject the `file://.` reference. Using `file://.` for a content path generated an error message stating `content filepath is tainted`. With this release, `oc-mirror` properly validates the '.' directory reference. (link:https://issues.redhat.com/browse/OCPBUGS-57970[OCPBUGS-57970])
+
+* Before this update, the `oc-mirror v2` plugin was not using the correct filtered catalog during its operations. As a consequence, unspecified operators were included in the configuration and the plugin tried to connect to the catalog registry during disk-to-mirror workflows, even in air-gapped environments. With this release, the correct filtered catalog is used. (link:https://issues.redhat.com/browse/OCPBUGS-57964[OCPBUGS-57964])
+
+* Before this update, modals that used the same `useModal` hook instance overwrote each other. As a consequence, the {product-title} Lightspeed user interface disappeared. With this release, modals have unique IDs. As a result, modal conflicts are resolved, which enables the simultaneous user interface display for the *Lightspeed*, *Troubleshooting Panel*, and *Networking* pages.  (link:https://issues.redhat.com/browse/OCPBUGS-57931[OCPBUGS-57931])
+
+* Before this update, images were ignored and recreated on every reconcile call, which caused new caches and prevented the use of cached images. As a consequence, memory usage of Hypershift quickly grew, which created  performance issues. With this release, images in Hypershift are cached effectively by creating and using a global registry provider instead of recreating registry and release providers on every reconcile call. As a result, memory usage is optimized in Hypershift. (link:https://issues.redhat.com/browse/OCPBUGS-57818[OCPBUGS-57818])
```

Comment:
```suggestion
* Before this update, images were ignored and recreated on every reconcile call, which caused new caches and prevented the use of cached images. As a consequence, memory usage of hosted control planes quickly grew, which created  performance issues. With this release, images in hosted control planes are cached effectively by creating and using a global registry provider instead of recreating registry and release providers on every reconcile call. As a result, memory usage is optimized in hosted control planes. (link:https://issues.redhat.com/browse/OCPBUGS-57818[OCPBUGS-57818])
```